src/main/java/net/huskycraft/blockyarena/listeners/EntityListener.java

```diff
@@ -37,19 +37,60 @@
 import org.spongepowered.api.entity.living.player.Player;
 import org.spongepowered.api.event.Listener;
 import org.spongepowered.api.event.cause.entity.damage.source.DamageSources;
+import org.spongepowered.api.event.command.SendCommandEvent;
 import org.spongepowered.api.event.entity.DamageEntityEvent;
+import org.spongepowered.api.event.filter.cause.First;
+import org.spongepowered.api.event.game.GameReloadEvent;
 import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.channel.MessageChannel;
+import org.spongepowered.api.text.chat.ChatTypes;
+import org.spongepowered.api.text.format.TextColors;
 import org.spongepowered.api.world.World;
 
 import java.util.Optional;
 
 public class EntityListener {
 
-    public static BlockyArena plugin;
 
-    public EntityListener(BlockyArena plugin) {
-        this.plugin = plugin;
+    public EntityListener() {
     }
+    
+    @Listener
+    public void onReload(GameReloadEvent event)
+    {
+    	BlockyArena.getInstance().reloadConfig();
+    }
+    
+	@Listener
+	public void onCommand(SendCommandEvent event, @First Player p) 
+	{
+		Gamer gamer = GamersManager.getGamer(p.getUniqueId()).get();
+
+		Game game = gamer.getGame();
+		
+		String command = event.getCommand();
+
+		if (game != null)
+		{
+			
+				if (game.getGameState() == GameState.RECRUITING || game.getGameState() == GameState.STARTED || game.getGameState() == GameState.STARTING || game.getGameState() == GameState.STOPPING) 
+				{
+						
+					MessageChannel.TO_ALL.send((Text)Text.builder("command issued : " + command).color(TextColors.GOLD).build());
```

Comment:
Do not broadcast commands issued by individual players. May leak sensitive information to others.

Reply:
Removed !
Was a test until I properly debug !